jsk_kinova_robot/README.md

```diff
@@ -69,9 +69,10 @@ mkdir -p ~/kinova_ws/src
 cd ~/kinova_ws/src
 wstool init
 wstool merge https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_kinova_robot/kinova.rosinstall
+wstool merge https://raw.githubusercontent.com/Kanazawanaoaki/jsk_robot/kinova-noetic/jsk_kinova_robot/kinova.rosinstall.${ROS_DISTRO} # Please ignore error if kinova.rosinstall.${ROS_DISTRO} does not exist. # We can remove this line after the https://github.com/ros/collada_urdf/pull/44 is merged
```

Comment:
I think it is okay with  
`https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/kinova-noetic/jsk_kinova_robot/kinova.rosinstall.${ROS_DISTRO}`

Reply:
I fixed it. https://github.com/jsk-ros-pkg/jsk_robot/pull/1906/commits/6022d95fb126b7b372855dca13eab0e2759c8ad1